src/content/docs/apm/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues.mdx

```diff
@@ -70,17 +87,37 @@ To enable Kafka event collection:
    ```
    SELECT average('producer-metrics.record-send-rate') from KafkaMetrics SINCE 30 minutes ago timeseries
    ```
+   If you are using Kafka Streams metrics, use the `KafkaStreamsMetrics` event type to access streams specific metrics.
+   
+## Enable Kafka Streams transactions [#collect-kafka-streams-transactions]
+If you are Kafka Streams, by default we do not enable transactions. This is to prevent unesessary overhead as Kafka applications tend to have high throughput. 
+
+Kafka Streams metrics unlike JMS metrics are not per record. Instead, a transaction begins when a Kafka consumer polls records and then the resulting data gets processed.
+
+If you do wish to create transactions, you need to enable a `kafka-streams-spans-<instrumentation module version number>` module.
+
+For example:
+
+```
+class_transformer:
+  com.newrelic.instrumentation.kafka-streams-spans-3.2.0:
+    enabled: true
+```
+
+Please refer to the [compatability docs](https://docs.newrelic.com/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent/) to see what instrumentation module you need to enable for your Kafka Streams version.
 
 ## Enable Kafka distributed traces [#collect-kafka-distributed-traces]
 
-The Java agent can also collect [distributed traces](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) from Kafka clients. Enabling traces doesn't affect the agent's normal operations: it will still report metric or event data from Kafka.
+The Java agent can also collect [distributed traces](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) from Kafka clients. Since Kafka Streams runs on top of Kafka Clients, the steps to manage distributed tracing also applies. 
```

Comment:
```suggestion
The Java agent can also collect [distributed traces](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) from Kafka clients. Since Kafka Streams runs on top of Kafka Clients, the steps to manage distributed tracing also apply. 
```